src/js/dashboard/dashboard.html

```diff
@@ -1,52 +1,48 @@
-<dashboard id="dashboard" class="row dashboard" ng-controller="DashboardComponent as dc">
-    <div class="columns small-12">
-        <div class="row cf bg-gray dashboard__header">
-            <div class="large-2 medium-4 small-12 columns bg-purple no-padding full-height">
-                <div class="row">
-                    <img class="dashboard__logo" src="/img/franklin-logo2.png" alt="logo">
-                    <p class="dashboard__title no-margin ">Franklin
-                        <br>Dashboard</p>
+<div class="dashboard" ng-controller="DashboardComponent as dc">
+    <div class="dashboard__nav">
+        <div class="row">
+            <div class="columns small-9" style="border-right: 1px solid #13a4ad;">
```

Comment:
Yeah I know, but was trying to separate Foundation columns from Franklin style, but because of foundation padding/margin, needed to be there, should I create a class and put it in the same markup?